### ISSUE TITLE
Code requests config.yaml when setting the --toml parameter.

### ISSUE THREAD

**Issue by tovare** [10Dimensional/hugo-algolia#7]
However, just copying the config.toml file to config.yaml and it seems to parse toml.

**Comment by lorenaramonda** [comment]
I guess the flag -toml only works for the front matter, but yes it seems that the entire module works only if there is a config.yaml and not a config.toml. I think this is a lack of design, it should work both with config.yaml and config.toml

**Comment by 10Dimensional** [comment]
Yeah, I built this pretty quickly when we were building out our docs. There was a little but of design involved, but the package was tailored for our needs. Good feedback, will get this fixed 👍
